src/dev-data/data/import-dev-data.js

```diff
@@ -0,0 +1,64 @@
+import mongoose from 'mongoose';
+import fs from 'fs';
+import Category from '../../models/CategoryModel.js';
+import User from '../../models/UserModel.js';
+import databaseConnect from '../../dbConnection.js';
+
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+
+// Function to import dev data
+const importData = async () => {
+
+  const users = JSON.parse(fs.readFileSync(`${__dirname}/users.json`, 'utf-8'));
+  const categories = JSON.parse(fs.readFileSync(`${__dirname}/categories.json`, 'utf-8'));
```

Comment:
Could you please move the code from lines 17 and 18 outside of the function? We only need to read the file once, but with the current implementation, the file is read every time the function is called, reducing our performance. I remember learning this best practice in Jonas' course

Reply:
let me fix that and try run again